typhos/cli.py

```diff
@@ -87,6 +87,14 @@
         'your application. Example: --size 1000,1000'
     ),
 )
+parser.add_argument(
+    '--hide-displays',
```

Comment:
Shortcut?
```suggestion
    "-h", "--hide-displays",
```

or maybe even
```suggestion
    "-M", "--minimal",
    dest="hide_displays",
```

Reply:
I think making the shortcut `-h` shadows the `-h`/`--help` and should be avoided.
Maybe just `--hide`? I'm not sold on `-m`/`--minimal`

Reply:
Just throwing out ideas, I'm not particularly fond of any of them, so the right answer is probably the lazy answer.

Reply:
And I'm throwing ideas back! It's a dialog. I'm also leaning toward lazy at this time.

Reply:
Unintentionally dismissive/salty sounding reply above, sorry! Thinking back, I still can't come up with anything better. I'd say this is good to go.